html/landpage.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+                                        <!-- LANDPAGE -->
```

Comment:
Maybe these comments could be at the beggining of the line (you can miss them right now).